tests/test_wrappers.py

```diff
@@ -157,6 +159,11 @@ def test_dict_observation_space_wrapper(env_spec):
     "env_spec", all_testing_env_specs, ids=[spec.id for spec in all_testing_env_specs]
 )
 def test_main_wrappers(wrapper, env_spec):
+    if (
```

Comment:
Could you add a comment for the future explaining why we are not testing in this case

Reply:
ok !